modules/microshift-olm-rh-ops-mirror.adoc

```diff
@@ -0,0 +1,14 @@
+
+//Module included in the following assemblies:
+//
+//* microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-olm-rh-ops-mirror_{context}"]
+= Using Red Hat-provided Operator catalogs and mirror registries
+
+You can filter and prune catalogs to get specific Operators and mirror them by using the oc-mirror {oc-first} plugin. You can also use Operators in disconnected settings or embedded in {op-system-ostree-first} images.
```

Comment:
If i am not wrong this is referring to oc-mirror v1 plugin as well, this needs to be modified.